src/map/anim/Map.PanAnimation.js

```diff
@@ -40,6 +40,10 @@ L.Map.include({
 	panBy: function (offset, options) {
 		offset = L.point(offset).round();
 		options = options || {};
+		this._animatingPan = true;
+		this.once('moveend', function () {
```

Comment:
Not sure about this way of resetting the flag actually, it may ends with race condition in some edge case. Should I just replace with resetting before everything `return this.fire.('moveend')` instead?